curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-067.md

```diff
@@ -0,0 +1,689 @@
+---
+id: 6148e5aeb102e3142de026a2
+title: Part 67
+challengeType: 0
+dashedName: part-67
+---
+
+# --description--
+
+Finally, the navigation accessibility can be improved with providing keyboard shortcuts.
```

Comment:
I had to look up how to use these keys. I know there's a lot of variations between browsers and OS's - but I think users will want to try it out after adding. So a couple of the major key combos or something might have been nice here.

Reply:
It is annoying how many different implementations there are, but I suppose that is web development.

Should we basically recreate the table on Mozilla, but leave out Opera and IE?: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey